server/app/views/dev/DatabaseSeedView.java

```diff
@@ -52,6 +57,33 @@ public Content render(
     String prettyActivePrograms = getPrettyJson(activePrograms);
     String prettyQuestions = getPrettyJson(questionDefinitions);
 
+    DivTag content =
+        div()
+            .with(h1(title))
+            .with(
+                div()
+                    .with(
+                        form()
+                            .with(makeCsrfTokenInputTag(request))
+                            .with(submitButton("index", "Go to dev database seeder page"))
+                            .withMethod("get")
+                            .withAction(routes.DevDatabaseSeedController.index().url())))
```

Comment:
**suggestion (blocking):** forms should not be used for `GET` requests, use a link (`<a>` tag) instead

Reply:
Applied `<a>` tag instead.